python_modules/automation/automation/docker/images/buildkite-integration-base/Dockerfile

```diff
@@ -159,3 +159,8 @@ RUN apt-get update -yqq \
     && apt-get install -yqq --no-install-recommends \
     libxmlsec1-dev \
     pkg-config
+
+# Install AWS CLI v2
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" \
```

Comment:
How did we use aws cli before (v1 or v2)? I know at least some jobs shelled out to it - were they managing their own installations?